### PR TITLE
Milestone4 - Implementing Calculator Logic

### PR DESCRIPTION
**Pull request by acushlakoncept**
In this PR, I implemented the application logic needed for the calculator app to function.
This includes the calculator.js and operate.js inside the logic folder.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  if (operation === '-') {
+    result = num1.minus(num2);
+  }
+  if (operation === '+') {
+    result = num1.plus(num2);
+  }
+  if (operation === 'X') {
+    result = num1.times(num2);
+  }
+  if (operation === '÷') {
```
Comment (mosaaleb) [inline src/logic/operate.js]: - [ ] Let's add another case for [`modifier`](https://mikemcl.github.io/big.js/#mod)

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Well done @acushlakoncept :clap:. You have handled most of the requirements, let's wrap it up by adding the `mod` case in `operate.js`. Check my comment below.

Please make the changes and submit a new code review.

##### Muhammad Ebeid

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: :clap: 
Hi @acushlakoncept :hand: 
Great job implementing the requested changes. Good luck with the next milestone.

**OPTIONAL**

- [ ] As it said in the requirements, the calculator width must be 700px. Your calculator width it's changing depending on the width of the screen. This is actually a mandatory requirement, but since this change was not specifically requested on milestone 3, I'm leaving as optional.
Screenshot from 2020-11-11 09-39-19

- [ ] The Button component should receive 2 props: color and wide. Color it's not required and, therefore, needs to have a default value of 'orange'. Wide prop needs to be required and true means the button should take 50% of the width, false should take 25%.
Screenshot from 2020-11-11 09-45-17
Screenshot from 2020-11-11 09-45-39

Happy Coding :computer: 
**Sergio Torres**

**Comment by acushlakoncept** [comment]
> ### Milestone approved 👍 👏
> Hi @acushlakoncept ✋
> Great job implementing the requested changes. Good luck with the next milestone.
> 
> **OPTIONAL**
> 
> * [ ]  As it said in the requirements, the calculator width must be 700px. Your calculator width it's changing depending on the width of the screen. This is actually a mandatory requirement, but since this change was not specifically requested on milestone 3, I'm leaving as optional.
>   Screenshot from 2020-11-11 09-39-19
> * [ ]  The Button component should receive 2 props: color and wide. Color it's not required and, therefore, needs to have a default value of 'orange'. Wide prop needs to be required and true means the button should take 50% of the width, false should take 25%.
>   Screenshot from 2020-11-11 09-45-17
>   Screenshot from 2020-11-11 09-45-39
> 
> Happy Coding 💻
> **Sergio Torres**

Thanks for the update, I will look into that, since in the current React Version (v1.1) Milestone 3 was kinda skipped, just went from milestone 2 to 4, I didn't get to add those requirements in 3.

Thank you.

**Pull request merged**